backend/routers/store_menus.go

```
@@ -45,48 +46,102 @@ func (s *Server) CreateMenu(c *gin.Context, storeID uuid.UUID) {
+	image, err := validators.ValidateRequireImageRequestBody(c, form.Image, bindErr, validators.ImageValidationConfig{
+		MaxImageSize:       maxImageSize,
+		MaxRequestBodySize: maxRequestBodySize,
+	})
+	if err != nil {
+		statusCode, message := validators.MapValidationErrorToHTTPStatusCode(err)
+		c.JSON(statusCode, ErrorResponse{
+			Message: message,
+		})
+		return
+	}
+	defer image.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `image.Close` is not checked

(errcheck)